test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -59,7 +59,8 @@ describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function() {
     expect(testContext.view.$('.retry-webauthn').css('display')).toBe('none');
     expect(testContext.view.$('.okta-waiting-spinner').css('display')).toBe('block');
     expect(testContext.view.$('.webauthn-not-supported').length).toBe(0);
-
+    expect(testContext.view.$('.js-cant-verify').length).toBe(1);
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('none');
```

Comment:
nit: Rather than having this in all tests I would just have a single test for this.